ui/src/components/MyPollsPreview/MyPollsPreview.css

```diff
@@ -0,0 +1,28 @@
+#previewBox {
+    width: 35%;
+    margin: 15px auto 15px;
+    padding: 5px;
+    border-radius: 10px;
+    display: block;
+    text-decoration: none;
+    text-align: left;
+}
+
+#pollName {
+    color: #FFFFFF;
```

Comment:
I'm not sure this `color` rule is necessary; make sure to try the code without the rule and see if it still has the same color. Black is the default color for text, and that's what is interesting here.

Reply:
The preview components have to be wrapped in <a> tags in order to be clickable, which defaults the colour to blue. The #FFFFFF rule is necessary to make it look like the Figma.